object/search.go

```diff
@@ -231,22 +241,27 @@ func (f *SearchFilters) addFlagFilter(keyTyp filterKeyType) {
 	f.addFilter(MatchUnknown, keyTyp, "", staticStringer(""))
 }
 
+// AddObjectVersionFilter adds a filter by version.
 func (f *SearchFilters) AddObjectVersionFilter(op SearchMatchType, v version.Version) {
 	f.addReservedFilter(op, fKeyVersion, staticStringer(version.EncodeToString(v)))
 }
 
+// AddObjectContainerIDFilter adds a filter by container id.
 func (f *SearchFilters) AddObjectContainerIDFilter(m SearchMatchType, id cid.ID) {
 	f.addReservedFilter(m, fKeyContainerID, id)
 }
 
+// AddObjectOwnerIDFilter adds a filter by user id.
```

Comment:
i think that "owner" is important work here. that it is an object's owner (not a container or any other owner)

Reply:
updated